examples/terraform-aws-s3-logging-enabled-example/README.md

```diff
@@ -0,0 +1,50 @@
+# Terraform AWS S3 Example
```

Comment:
Oh I feel bad saying this after all the effort into creating a new example, but I actually meant that you should update the existing example as opposed to creating a new one (as suggested here: https://github.com/gruntwork-io/terratest/pull/689#discussion_r518288068)

This is more an art instead of a policy, but the main reason for it is that most people will look at the S3 example for inspirations on testing s3 buckets, and it would be a bit of a sprawl to have a separate example/test just for the logging test (since it is a relatively small extension of the existing s3 example).

We want to limit the number of examples/tests we add because the overhead of another example is not just UX, but it also affects our testing infra as well, as that is more code to maintain when we do, for example, a terraform version update.

Again this is a bit difficult to explain given that it is more an art of balancing tradeoffs, but in this case I think it would be favorable to add access logging to the existing s3 example and add the additional check to the existing test.

Reply:
gotcha! no worries - it's all good experience! 

Having less code to maintain is a really good reason that resonates, and also giving people less variations will also be more straightforward! Agree :)

Reply:
It looks like you still have the new example here. Was there some discussion where we decided to go with the new example instead of consolidating?

Reply:
oh no - that's a mistake! correcting it now..

Reply:
D'oh! I thought I pushed - but apparently hadn't done so. All pushed up now :) My bad!